core/controlplane/config/templates/cloud-config-controller

```diff
@@ -760,6 +760,7 @@ write_files:
 
       # Configmaps
       kubectl apply -f "${mfdir}/kube-dns-cm.yaml"
+      kubectl apply -f "${mfdir}/kube-proxy-config.yaml"
```

Comment:
Can you use a style more consistent with the other objects, like this:

```bash
# ConfigMaps
for manifest in {kube-dns,kube-proxy}; do
    kubectl apply -f "${mfdir}/$manifest-cm.yaml"
done
```
.. and use the `-cm` suffix in the ConfigMap file, instead of `-config`?

Reply:
Good idea, changed.